estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,35 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.tools import relativedelta
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Real Estate Property Offer"
+
+    price = fields.Float()
+    status = fields.Selection([
+        ('accepted', 'Accepted'),
+        ('refused', 'Refused'),
+    ], copy=False)
+
+    partner_id = fields.Many2one('res.partner', required=True)
+    property_id = fields.Many2one('estate.property', required=True)
+
+    validity = fields.Integer(string="Offer Validity (days)", default=7)
+    date_deadline = fields.Date(string="Deadline", compute="_compute_deadline_date", inverse="_inverse_deadline_date")
+
+    @api.depends('validity')
```

Comment:
I know it is confusing because `create_date`is never supposed to change but you still have to tell the ORM that this method depends on it because it is used in the computation logic